kube_controller_manager/datadog_checks/kube_controller_manager/kube_controller_manager.py

```diff
@@ -149,6 +151,13 @@ def __init__(self, name, init_config, instances):
 
                 instance['health_url'] = url
 
+                slis_instance = self.create_sli_prometheus_instance(instance)
+                instance['sli_scraper_config'] = self.get_scraper_config(slis_instance)
+                if instance.get('slis_available', None) is None:
```

Comment:
```suggestion
                if instance.get('slis_available') is None:
```

Non blocking nit. The default return for `.get` is `None` if the key isn't found